python-sdk/tests/files/locations/test_ftp.py

```diff
@@ -0,0 +1,54 @@
+import ftplib
+from unittest.mock import Mock, patch
+
+from airflow.providers.ftp.hooks.ftp import FTPHook
+
+from astro.files.locations import create_file_location
+
+
+def test_get_transport_params_for_ftp():  # skipcq: PYL-W0612, PTC-W0065
+    """test get_transport_params() method which should empty dict"""
+    path = "ftp://bucket/some-file"
+    location = create_file_location(path)
+    credentials = location.transport_params
+    assert credentials == {}
+
+
+@patch("airflow.providers.ftp.hooks.ftp.FTPHook.get_size")
+def test_size(mock_get_conn):
+    """Test get_size() of for FTP file."""
```

Comment:
```suggestion
    """Test whether get_size() will return the file size correctly """
```

Reply:
you are mocking the response I think it better to check has been called with correct param or not using assert_call_once_with(...)

Reply:
modified to assert_once_called with and also docstring